### PR TITLE
Implement theme switch

### PR DESCRIPTION
**Pull request by mr-pavlenko**
- Add theme switcher
- Adjust Header and Footer for two themes (light and dark)
- Adjust Transactions, Blocks, Validators and Top Account pages for two themes

### CODE REVIEW

File: src/components/AppPagination/AppPagination.vue [hunk 1]
```diff
@@ -157,18 +154,16 @@ function goToSelectedPage(event: { target: { value: string } }): void {
...
-  background: var(--clr__secondary);
   border-radius: 0.4rem;
 }
 
 .app-pagination__input {
   width: 6rem;
   height: 3.6rem;
   margin: 0 0.2rem;
-  border: none;
+  border: 1px solid var(--clr__input-border);
```
Comment (nikita1tup) [inline src/components/AppPagination/AppPagination.vue:164]: rem

File: src/components/BackButton.vue [hunk 2]
```diff
@@ -1,9 +1,18 @@
...
+      height="16"
+      viewBox="0 0 16 16"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M16 7H3.83L9.42 1.41L8 0L0 8L8 16L9.41 14.59L3.83 9H16V7Z"
+        fill="#ffffff"
+      />
+    </svg>
```
Comment (nikita1tup) [inline src/components/BackButton.vue:14]: mb will be better create vue component with SVG

File: src/styles/tables.scss [hunk 3]
```diff
@@ -16,7 +16,7 @@
   }
 
   &__head {
```
Comment (nikita1tup) [inline src/styles/tables.scss:18]: remove waterfalls in this file

File: src/components/ThemeSwitch.vue [hunk 4]
```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="theme-switch">
+    <button
+      :class="[
+        'theme-switch__btn',
+        currentTheme === ThemeMode.Dark ? 'theme-switch__btn--active' : '',
```
Comment (nikita1tup) [inline src/components/ThemeSwitch.vue:6]: create computed variable and check isLight, for dark mode 'theme-switch__btn--active' : !isLight , for light  'theme-switch__btn--active' : isLight

File: src/components/ThemeSwitch.vue [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    </button>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { ref } from 'vue'
+import { ThemeDark, ThemeLight } from '@/components/icons'
+import { Theme, ThemeMode } from '@/helpers/theme'
+
+const currentTheme = ref<string>(Theme.getTheme())
```
Comment (nikita1tup) [inline src/components/ThemeSwitch.vue:29]: ref(Theme.getTheme()) is ok

**Review by nikita1tup — CHANGES_REQUESTED** [review]

File: src/components/AppPagination/AppPagination.vue (hunk 1 above)
Comment (mr-pavlenko) [reply src/components/AppPagination/AppPagination.vue]: done

File: src/components/BackButton.vue (hunk 2 above)
Comment (mr-pavlenko) [reply src/components/BackButton.vue]: Moved to the separate icon component

File: src/styles/tables.scss (hunk 3 above)
Comment (mr-pavlenko) [reply src/styles/tables.scss]: Done for the whole file

File: src/components/ThemeSwitch.vue (hunk 4 above)
Comment (mr-pavlenko) [reply src/components/ThemeSwitch.vue]: Done

File: src/components/ThemeSwitch.vue (hunk 5 above)
Comment (mr-pavlenko) [reply src/components/ThemeSwitch.vue]: Ok, done

**Pull request merged**